lib/chef/resource/rhsm_register.rb

```diff
@@ -27,43 +27,35 @@ class RhsmRegister < Chef::Resource
                   " or a local Red Hat Satellite server."
       introduced "14.0"
 
-      property :activation_key,
-               [String, Array],
+      property :activation_key, [String, Array],
                coerce: proc { |x| Array(x) },
                description: "A String or array of the activation keys to use when registering. You must also specify the organization property if using activation_key."
 
-      property :satellite_host,
-               String,
+      property :satellite_host, String,
                description: "The FQDN of the Satellite host to register with. If not specified, the host will be registered with Red Hat's public RHSM service."
 
-      property :organization,
-               String,
+      property :organization, String,
                description: "The organization to use when registering, required when using an activation key"
 
-      property :environment,
-               String,
+      property :environment, String,
                description: "The environment to use when registering, required when using username and password"
 
-      property :username,
-               String,
+      property :username, String,
                description: "The username to use when registering. Not applicable if using an activation key. If specified, password and environment are also required."
 
-      property :password,
-               String,
+      property :password, String,
                description: "The password to use when registering. Not applicable if using an activation key. If specified, username and environment are also required."
 
       property :auto_attach,
                [TrueClass, FalseClass],
```

Comment:
Why not move this one up to the property line?

Reply:
I wasn't breaking them up until I added the descriptions. That bloats the lines so much it's easier to read on individual lines.